### PR TITLE
Add compliance with Swift code generation

### PR DESCRIPTION
**Pull request by TSC21**

### CODE REVIEW

File: protos/custom_action/custom_action.proto [hunk 1]
```diff
@@ -105,12 +105,12 @@ message ActionToExecute {
 // sent and executed during a custom action.
 message Command {
     // Command type enumeration
-    enum Type {
-        TYPE_LONG = 0; // Command long
-        TYPE_INT  = 1; // Command int
+    enum CommandType {
```
Comment (JonasVautherin) [inline protos/custom_action/custom_action.proto:108]: Was that change necessary?

File: protos/custom_action/custom_action.proto [hunk 2]
```diff
@@ -146,8 +146,8 @@ message Stage {
 // Metadata that describes the custom action and defines its stages.
 message ActionMetadata {
     uint32 id = 1; // ID of the action
-    string name = 2; // Name of the action
-    string description = 3; // Description of the action
+    string action_name = 2; // Name of the action
```
Comment (JonasVautherin) [inline protos/custom_action/custom_action.proto:149]: Those changes are not necessary either, I would think

File: protos/obstacle_avoidance_server/obstacle_avoidance_server.proto [hunk 3]
```diff
@@ -24,21 +24,21 @@ service ObstacleAvoidanceServerService {
...
     // Possible obstacle avoidance service control commands, according to
     // MAVLink COMPONENT_CONTROL enum.
-    enum Type {
-        TYPE_UNKNOWN = 0; // Unknown control command.
-        TYPE_START = 1; // Start/turn-on obstacle avoidance service.
-        TYPE_STOP = 2; // Stop/turn-off obstacle avoidance service.
-        TYPE_RESTART = 3; // Restart/reboot obstacle avoidance service.
-        TYPE_ENABLE = 4; // Enable obstacle avoidance service. Used to switch the service from an idle state to an active state.
-        TYPE_DISABLE = 5; // Disable obstacle avoidance service. Used to switch the service from an active state to an idle state.
+    enum ControlType {
```
Comment (JonasVautherin) [inline protos/obstacle_avoidance_server/obstacle_avoidance_server.proto:34]: I'm a bit confused on why the previous version did not work, too :thinking:

**Review by JonasVautherin — APPROVED** [review]

File: protos/custom_action/custom_action.proto (hunk 2 above)
Comment (TSC21) [reply protos/custom_action/custom_action.proto:149]: the description one was

File: protos/obstacle_avoidance_server/obstacle_avoidance_server.proto (hunk 3 above)
Comment (TSC21) [reply protos/obstacle_avoidance_server/obstacle_avoidance_server.proto:34]: bit painful yes lol
